src/iop/CMakeLists.txt

```diff
@@ -142,7 +142,7 @@ endif(GMIC_FOUND)
 add_iop(toneequal "toneequal.c" DEFAULT_VISIBLE)
 add_iop(filmicrgb "filmicrgb.c")
 add_iop(negadoctor "negadoctor.c")
-add_iop(channelmixerrgb "channelmixerrgb.c")
+add_iop(channelmixerrgb "channelmixerrgb.c" "../chart/common.c")
```

Comment:
This looks wrong. The second parameter is a flag (DEFAULT_VISIBLE) to set the default visibility of the module. I don't think you need to add "../chart/common.c" here at all as it is not an iop. If the file is to be compiler it should be added into src/CMakeFile.list.

Reply:
It is ok, I was wrong sorry for the noise.